akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala

```diff
@@ -77,21 +78,25 @@ class ManualTimerExampleSpec extends ScalaTestWithActorTestKit(ManualTime.config
 
       val behavior = Behaviors.withTimers[Command] { timer =>
         timer.startPeriodicTimer("T", Tick(1), interval)
-        Behaviors.receive { (context, cmd) =>
-          cmd match {
-            case Tick(n) =>
-              probe.ref ! Tock(n)
-              Behaviors.same
-            case SlowThenBump(nextCount) =>
-              manualTime.timePasses(interval)
-              timer.startPeriodicTimer("T", Tick(nextCount), interval)
-              probe.ref ! SlowThenBumpAck
-              Behaviors.same
-          }
+
+        probe.ref ! Started
+        Behaviors.receiveMessage {
+          case Tick(n) =>
+            probe.ref ! Tock(n)
+            Behaviors.same
+          case SlowThenBump(nextCount) =>
+            manualTime.timePasses(interval)
+            timer.startPeriodicTimer("T", Tick(nextCount), interval)
+            probe.ref ! SlowThenBumpAck
+            Behaviors.same
         }
       }
 
       val ref = spawn(behavior)
+
+      // make sure we actually started the timer before we change the time
+      probe.expectMessage(Started)
```

Comment:
This is the crux right? The rest is mostly formatting and minors.

Reply:
Yep! Compiler warnings mostly, the rest.